install-from-binstall-release.ps1

```diff
@@ -0,0 +1,20 @@
+$tmpdir = $Env:TEMP
```

Comment:
Is there anything like `set -euxo pipefail` in powershell?
I think it might be the problem why this script fails silently.

Reply:
It seems `set -e` can be done via `$ErrorActionPreference = "Stop"`.

I'm not sure how to error when a variable is missing (`set -u`) but every variable used (apart from Path and HOME in the final check) is defined within the script, so it should be fine?

I think `set -x` can be done with `Set-PSDebug -Trace 1`.

I don't use any pipes in the script, so I don't think `pipefail` is needed either.

Reply:
It's good as long as all potential errors are covered.